### ISSUE TITLE
Navigation react-router-dom not working

### ISSUE THREAD

**Issue by durdevic** [fxnoob/chrome-extension-boilerplate#3]
Hey man, I tried using your plugin and I wanted to create Login/Home switch routes with **react-router-dom** but unfortunately it's not working. 

Do you have an idea how can I do that. The goal is to get the tokens on the Login page, and log out when the tokens expire. 

In index.js I created an App component
```
class App extends Component {
  render() {
    return (
      <ApolloProvider client={client}>
        <Routes></Routes>
      </ApolloProvider>
    );
  }
}
```

and my Routes.js looks like this
```
import React, { PureComponent } from "react";
import { BrowserRouter, Switch, Route, Redirect } from "react-router-dom";

import Login from "./components/Login";
import Home from "./components/Home";

export class Routes extends PureComponent {
  render() {
    return (
      <BrowserRouter>
        <Switch>
          <Route exact path="/" render={() => <Redirect to="/login" />} />

          <Route path="/login" component={Login} />
          <Route path="/home" component={Home} />
        </Switch>
      </BrowserRouter>
    );
  }
}

export default Routes;

```

He is just showing me the empty page. 

Btw I had a problem with an error **regeneratorRuntime is not defined**, and I solved it by adding these to the .babelrc

```
npm install --save @babel/runtime 
npm install --save-dev @babel/plugin-transform-runtime
```

```
{
  "presets": [
    "@babel/preset-env",
    "@babel/preset-react"
  ],
  "plugins": [
    "@babel/plugin-proposal-class-properties" ,
    "@babel/plugin-syntax-class-properties",
    "@babel/plugin-syntax-dynamic-import",

    "@babel/plugin-transform-runtime"
  ]
}
```

The issue is described here 
https://stackoverflow.com/a/53736090

**Anyway really good boilerplate till now. Thank you for it! :)**

**Comment by fxnoob** [comment]
Hi @durdevic ,
I had the same issues when I was working on the similar thing in https://github.com/fxnoob/paytm-history-extension/blob/master/option-page/components/index.jsx.
I could not directly access the declared route except the main route. While navigating to the other routes from main routes was only possible thing.

**Comment by fxnoob** [comment]
issue resolved in 375d2b88cb9967004b7fb14e2a6cd1bee7797968.
Thanks.
